plugin/kubernetes/handler.go

```diff
@@ -29,7 +29,7 @@ func (k Kubernetes) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.M
 
 	switch state.QType() {
 	case dns.TypeAXFR, dns.TypeIXFR:
-		k.Transfer(ctx, state)
+		k.TransferDeprecated(ctx, state)
```

Comment:
not worth the effort imo. I *think* the entire case can be dropped and the plugin does the right thing